packages/rocketchat-ui/client/lib/chatMessages.js

```diff
@@ -554,7 +554,13 @@ this.ChatMessages = class ChatMessages {
 	}
 
 	isMessageTooLong(message) {
-		return message && message.length > this.messageMaxSize;
+		const adjustedMessage = message.replace(/:\w+:/gm, (match) => {
+			if (RocketChat.emoji.list[match] !== undefined) {
+				return ' ';
+			}
+			return match;
+		});
+		return message && RocketChat.messageProperties(adjustedMessage).length > this.messageMaxSize;
```

Comment:
this test on 'message' should not be done before?

Reply:
Yeah, sorry about that :sweat_smile: